matomo/content.md

```diff
@@ -68,4 +68,6 @@ We'd love to hear your feedback and suggestions in the issue tracker: [github.co
 
 ## GeoIP
 
-This product includes GeoLite data created by MaxMind, available from [http://www.maxmind.com](http://www.maxmind.com).
+~~This product includes GeoLite data created by MaxMind, available from [http://www.maxmind.com](http://www.maxmind.com).~~
```

Comment:
Maybe this should have an example of how to get it or at least show where to put it in the image (embed/improve part of the commented out portion of the Dockerfile)?

Reply:
I can link to https://matomo.org/faq/how-to/faq_163/, but this issue is not yet documented.